### ISSUE TITLE
chore: Create test accounts for project submission

### ISSUE THREAD

**Issue by Curtis-Thomas** [group13-Advanced-Web-Project/Movie-Central#204]
Test accounts should be created which will be added to the submission.

Minimum:

1 x Profile role - Admin
1 x Group Role - Admin
2 x Profile role - User

Usernames, passwords, and account accessibility descriptions should be provided.
